src/rdkafka_cgrp.c

```diff
@@ -3835,8 +3873,8 @@ rd_kafka_error_t *rd_kafka_consumer_group_metadata_read (
                         RD_KAFKA_RESP_ERR__BAD_MSG,
                         "Input buffer has invalid stop byte");
 
-        /* We now know that group_id is printable-safe and is nul-terminated. */
-        *cgmdp = rd_kafka_consumer_group_metadata_new(group_id);
+        /* We now know that group_id is printable-safe and is null-terminated. */
```

Comment:
Please revert.
Ascii `\0` is called Nul, not Null.

This nomenclature helps disambiguating between NULL pointers (4-8 bytes) and NUL (1 byte).

Reply:
didn't fully appreciate that, reverting. that said, according to wikipedia, i think null is probably more correct in written english.